modules/file-input/main/index.coffee

```diff
@@ -39,14 +49,15 @@ class FileInput extends hx.EventEmitter
       multiple: false
       dragEnabled: true
       buttonClass: 'hx-action'
-      noFilesText: 'No File Chosen'
-      filesSelectedText: 'Files Selected'
-      buttonText: 'Choose File'
+
+      buttonText: hx.userFacingText('fileInput', 'chooseFile')
```

Comment:
I was more getting at the use of `buttonText`/`chooseFile` for the same option (depending on if you set it via `hx.userFacingText` or the `options` object).

However backwards compatibility is obviously needed, so just something to revisit in 2.0.0..